### ISSUE TITLE
The "Error: unexpected end of file" error on https://www.fiverr.com/

### ISSUE THREAD

**Issue by LavrovArtem** [DevExpress/testcafe-hammerhead#1915]
```
{ Error: unexpected end of file
    at Gunzip.zlibOnError (zlib.js:153:15) errno: -5, code: 'Z_BUF_ERROR' }
```
Site: https://www.fiverr.com/

**Comment by LavrovArtem** [comment]
I think it is the same issue as:
https://github.com/nodejs/node/issues/8701#issuecomment-268447734

**Comment by LavrovArtem** [comment]
fixed in #1916
